src/components/modal/modal.tsx

```diff
@@ -320,6 +343,12 @@ export class Modal
 
   contentId: string;
 
+  @State() modalSlottedWidth: number;
+
+  @State() cssWidth: string | number;
```

Comment:
these should be css vars

Reply:
First one was stale code - removed it. The other two are set based on the css vars for use in `renderStyles()`.